nuxt.config.ts

```diff
@@ -66,8 +66,8 @@ export default defineNuxtConfig({
     public: {
       // seoKit
       siteUrl: PUBLIC_SITE_URL,
-      siteName: 'IPFS Powers the Distributed Web',
-      siteDescription: 'The InterPlanetary File System is a peer-to-peer hypermedia protocol designed to preserve and grow humanity\'s knowledge by making the web upgradeable, resilient, and more open.',
+      siteName: 'IPFS is an open system to manage data without a central server',
```

Comment:
This key drives for the primary <title> tag for the site used on all pages so I would suggest something shorter, perhaps even simply `IPFS` (however I do prefer the current descriptive title):

<img width="642" alt="2023-09-22 @ 22 43 15@2x" src="https://github.com/ipfs/ipfs-website/assets/106938/5102a7fd-c47d-45fe-8c60-d5a4ed9c699a">

Reply:
What do you think about:

"IPFS – open system to manage data without a central server"

That's 58 characters which won't get truncated and still follows the under 60 characters SEO rule of thumb.

[source](https://moz.com/learn/seo/title-tag#:~:text=Keep%20title%20tags%20between%2051,or%20rewrite%20it%20for%20display.)

Reply:
What about taking this change only for the homepage and on the other pages just do: `Developers | IPFS"?

Reply:
Any thoughts? @cwaring @lidel

Reply:
If we go for a different title on the index page and sub pages then the index title would end up looking something like this: `An open system to manage data without a central server | IPFS`

Reply:
If that sounds ideal happy to make the change now

Reply:
Sounds good -- let's go with that

Reply:
updated in 724256b